src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.Reflection.cs

```diff
@@ -129,28 +129,40 @@ private bool IsType(XamlType xamlType, ISymbol typeSymbol)
 		{
 			var type = FindType(xamlType);
 
-			if (type != null)
+			return IsType(type, typeSymbol);
+		}
+
+		private bool IsType(INamedTypeSymbol symbol, ISymbol typeSymbol)
```

Comment:
The parameter names here are a bit confusing (particularly compared to their respective types)